buildKonfig.properties

```diff
@@ -0,0 +1,8 @@
+# Staging env
+STAGING_CLIENT_ID=R9v5TscbY-_gxCtcmeJHIPm5tzXKkopGSsjUU7Hqg9M
```

Comment:
@AVI5HEK Please remove this file

Reply:
How was it added! It was included in .gitignore! Gonna remove and force-push.

Reply:
Removed.